File: app/src/main/java/com/news/gson/NewsList.java
package com.news.gson;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class NewsList {

    public int code;

    public String msg;

    @SerializedName("newslist")//json数组名需要序列化
    public List<News> newsList;

}

File: app/src/main/java/com/news/gson/NewsList.kt
package com.news.gson

import com.google.gson.annotations.SerializedName

class NewsList {

    var code: Int = 0

    var msg: String? = null

    @SerializedName("newslist")//json数组名需要序列化
    var newsList: List<News>? = null

}
